android/src/main/java/com/getcapacitor/community/admob/AdMob.java

```diff
@@ -65,12 +65,26 @@ public void onInitializationComplete(InitializationStatus initializationStatus)
                 }
             );
             bannerExecutor.initialize();
+
             call.resolve();
         } catch (Exception ex) {
             call.reject(ex.getLocalizedMessage(), ex);
         }
     }
 
+    @PluginMethod
+    public void globalSettings(final PluginCall call) {
+        if (call.getFloat("volume") != null) {
```

Comment:
*Nice To have but not needed:*

Eventually `volume` and `muted` can be put into an enum and create a recursive function that parses that enum and see if any of those values are defined in the pluginCall. Right now it is really too much work for just 2 values thought.

Reply:
`volume: 0.0` and `muted: true`  seem to be different working.

```
volume: 0.7 → muted: true → muted: false : volume: 0.7 is auto set.
```

I think this should be left to the user to use.

I agree volume must be enum .